tests/nightly/test_large_array.py

```diff
@@ -86,8 +86,8 @@ def test_ndarray_random_randint():
 def test_ndarray_random_exponential():
     scale_array = nd.random.uniform(shape=(MEDIUM_X, SMALL_Y))
     a = nd.random.exponential(scale=scale_array, shape=(SMALL_X, SMALL_Y))
-    assert a.shape == (MEDIUM_X, SMALL_Y, SMALL_X, SMALL_Y)
     assert a[-1][0][0][0] >= 0
+    assert a.shape == (MEDIUM_X, SMALL_Y, SMALL_X, SMALL_Y)
```

Comment:
any particular reason for switching the order of this line?

Reply:
Yes. Updated in description.
Shape evaluation assert passes for large tensor (despite it actually supposed to fail because it doesn't compute shape unless the ndarray is really needed to be accessed)

hence by first accessing value of ndarray, we are effectively forcing to compute shape and then assert it.

Makes sense?

Reply:
But to keep it consistent with rest of the ops that have been tested, switching makes sense in that regard